volmdlr/edges.py

```diff
@@ -2597,6 +2666,54 @@ def point_at_abscissa(self, curvilinear_abscissa):
         # # Outside of length
         # raise ValueError
 
+    def norm_eval_at_point(self, point: volmdlr.Point3D, resolution=20):
```

Comment:
is it the same than normal for other edges? https://github.com/Dessia-tech/volmdlr/blob/1304748eb8d41dd3c1b6220a0fb9d5a7659ef27d/volmdlr/edges.py#L2592

Reply:
'norm_eval_at_point' is short for 'normalized_evaluation_at_point', and search for the value of the normalized evaluation parameter t (between 0 and 1) that would return the given point when the BSplineCurve3D is evaluated at the t value.
Should I change the name of the function for something more explicit ?

Reply:
Can you put it in docstring?